src/test/java/InfrastructureManager/AdvantEdge/AdvantEdgeClientTests.java

```diff
@@ -53,6 +56,21 @@ public void deployScenarioRequestTest() {
         verify(postRequestedFor(urlEqualTo(path)));
     }
 
+    @Test
+    public void updateNetworkCharacteristics() throws IOException {
+        String sandboxName = "test";
+        String path = "/" + sandboxName + "/sandbox-ctrl/v1/events/NETWORK-CHARACTERISTICS-UPDATE";
+        String jsonTestPath = "src/test/resources/AdvantEdge/network-update-test.json";
+        client.out("advantEdge networkUpdate " + sandboxName + " fog-1 FOG 10 10 50 1 0");
```

Comment:
these numbers should match the ones in [the test file](https://github.com/EdgeLab-FHDO/Edge-Diagnostic-Platform/pull/39/files#diff-56b91ca81e0002da15b476b51c7f14b2)